lsm-tree.cabal

```diff
@@ -167,7 +168,7 @@ library
     , lsm-tree:bloomfilter
     , lsm-tree:kmerge
     , lsm-tree:monkey
-    , nothunks              ^>=0.2
+    , nothunks              ^>=0.3
```

Comment:
To be consistent with other testing related stuff, shouldn't this dependency and instances be defined in test component only?

Reply:
Hmm, yes that could make sense. It does mean we don't get nothunks checks "for free" with assertions enabled, the tests would have to make those assertions explicitly. Maybe that's fine